src/simsopt/mhd/vmec.py

```diff
@@ -752,7 +752,11 @@ def run(self):
 
             # Delete the previous output file, if desired:
             for filename in self.files_to_delete:
-                os.remove(filename)
+                try:
+                    os.remove(filename)
+                except FileNotFoundError:
+                    logger.debug('Tried to delete the file {filename} but it was not found')
```

Comment:
I think you forgot an f-string here ;)

Reply:
Definitely! Thanks a ton. I committed a fix.